app/models/book.rb

```diff
@@ -1,2 +1,7 @@
 class Book < ApplicationRecord
+  belongs_to :categories, class_name: Category.name,
+                          foreign_key: :category_id,
+                          optional: true
```

Comment:
```suggestion
  belongs_to :category
```
ri chứ em
ngoài ra, 1 sách phải thuộc thể loại nào chứ, sao lại có `optional true ` dc bé